bfillings/vsearch.py

```diff
@@ -0,0 +1,561 @@
+#!/usr/bin/env python
```

Comment:
The shebang is not needed because this is a library code, and is not tailored to be executed through the CLI